.github/workflows/build.yml

```diff
@@ -0,0 +1,98 @@
+# **what?** - This workflow builds python packages and ensures they can be
+# installed on support OS and python versions. This workflow should not require
+# any secrets.
+# **why?** - Ensure that dbt and its dependencies are able to be packaged
+# and installed.
+# **when?** - This will run when pushing to protected branches, pull requests,
+# and manual dispatch.
+
+name: Build
+
+on:
+  push:
+    branches:
+      - "main"
+      - "develop"
+      - "*.latest"
+      - "releases/*"
+  workflow_dispatch:
+  pull_request:
+
+concurrency:
+  group: build-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build python
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
There is no matrix for python version in this job and below you hard code 3.8 as the version. Are you only building with 3.8 and if so then this can be removed

Reply:
ah ya good call!